admin/tabs/settings-config.php

```diff
@@ -35,8 +35,7 @@ public function content() {
     }
 
     public function main_column() {
-        global $wpdb;
-        $setting = get_option( 'dt_setup_wizard_config' );
+        $setting = json_encode( get_option( 'dt_setup_wizard_config' ), JSON_PRETTY_PRINT );
```

Comment:
Can you update the flags passed into this to be: `JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES`. All of the slashes in URLs were being escaped which is confusing and hard to read, and it would probably save wrong.